src/stores/business.ts

```diff
@@ -16,7 +16,16 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
 
   const currentBusinessAddresses: Ref<EntityAddressCollectionI> = ref(undefined)
   const currentParties: Ref<PartiesI> = ref(undefined)
+
   const currentBusinessIdentifier = computed((): string => currentBusiness.value?.identifier)
+
+  // set BUSINESS_ID session storage when business identifier is loaded
+  watch(currentBusinessIdentifier, (value) => {
```

Comment:
If we have a singular place (like store or API call) where we are loading this it would be better to set it there without watch.
Otherwise, if value is being set in multiple places, this is good place as any other.